AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -97,8 +100,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable id<ASLayoutable>)childForIdentifier:(NSString *)identifier;
 
-/** Returns all children added to this layout spec. */
-- (NSArray<id<ASLayoutable>> *)children;
+/**
+ * Returns all children added to this layout spec.
+ */
+- (nullable NSArray<id<ASLayoutable>> *)children;
```

Comment:
@appleguy I added the nullable annotation in this case as:
1. If no children where assigned we will return nil in this case
2. We nil out the children's array in the setChildren: method now before checking if children where provided and only than assign a newly created array

Any thoughts in general about it?